static/schemas/2.0/node.2.0.json

```diff
@@ -47,6 +47,10 @@
                         "type": "string",
                         "uniqueItems": true
                     }
+                },
+                "identifiers": {
+                    "description": "Node Macs",
+                    "type": "array"
```

Comment:
The identifier is a list of string, so copy the property `tags`, and the sub schema for each array item:
```
items: {
  type: "string"
}
```